adapter/internal/ga/client.go

```diff
@@ -46,7 +44,8 @@ import (
 var (
 	// apiRevision Map keeps apiUUID -> revisionUUID. This is used only for the communication between global adapter and adapter
 	// The purpose here is to identify if the certain API's revision is already added to the XDS cache.
-	apiRevisionMap map[string]*ga_model.Api
+	apiRevisionMap    map[string]*ga_model.Api
+	apiEnvRevisionMap = make(map[string]map[string]*ga_model.Api)
```

Comment:
Let's add a comment just like in line 45 to indicate what is first key, and second key and finally the value.

Reply:
addressed via https://github.com/wso2/product-microgateway/commit/ae2c06882353bda7a59101381ef3e82f88667a21